docs/getting-started/install-scylla/oss-install-azure.rst

```diff
@@ -0,0 +1,99 @@
+.. |VERSION| replace:: 5.2
+.. |IMAGE-DEFINITION| replace:: scylla-5.2
+.. |IMAGE-VERSION| replace:: 5.2.1
+.. |GALLERY-NAME| replace:: 6c268694-47ab-43ab-b306-3c5514bc4112
+
+==========================
+Launch ScyllaDB on Azure
+==========================
+
+This article will guide you through self-managed ScyllaDB deployment on Azure. For a fully-managed deployment of ScyllaDB 
+as-a-service, see `ScyllaDB Cloud documentation <https://cloud.docs.scylladb.com/>`_.
+
+With the right setting, ScyllaDB image can be used for production. For the ScyllaDB Image documentation, see 
+the `ScyllaDB Machine Image project <https://github.com/scylladb/scylla-machine-image>`_.
+
+.. note::
+    The article covers launching a ScyllaDB image from CLI. As an alternative, you can launch a ScyllaDB instance from 
+    the `Azure VM portal <https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.Compute%2FVirtualMachines>`_ -
+    search for *ScyllaDB Image* in Community Images.
+
+Prerequisites
+----------------
+
+* Active Azure account
+* `Azure CLI <https://learn.microsoft.com/en-us/cli/azure/install-azure-cli>`_
+* ScyllaDB Image requires at least 2 vCPU servers.
+
+ScyllaDB |VERSION| Image Information
+---------------------------------------
+
+* gallery-image-definition: |IMAGE-DEFINITION|
+* gallery-image-version: |IMAGE-VERSION|
+* public-gallery-name: |GALLERY-NAME|
```

Comment:
I do not want to update the doc for each **patch** release.
For now, lets do the same as you did in AWS, and point users to the website.
Later, we will have a dedicated mini doc site for cloud images